d3a_interface/utils.py

```diff
@@ -182,3 +182,44 @@ def get_area_name_uuid_mapping(serialized_scenario, mapping={}):
             new_mapping = get_area_name_uuid_mapping(child, mapping)
             mapping.update(new_mapping)
     return mapping
+
+
+def round_floats_for_ui(number):
+    return round(number, 3)
+
+
+def create_subdict_or_update(indict, key, subdict):
```

Comment:
Duplicate of `create_or_update_subdict` right?

Reply:
Not really, `create_subdict_or_update`creates an empty dict if key not found, whereas `create_or_update_subdict ` creates a dict with `subdict` if key is not found